tests/unittest/_torch/speculative/hw_agnostic/test_mtp.py

```
@@ -1742,18 +1742,16 @@ def test_prepare_drafter_inputs(
-        ("Gemma4ForCausalLM", True, True),
-        ("Gemma4ForConditionalGeneration", False, False),
-        ("LlamaForCausalLM", True, False),
+        ("Gemma4ForCausalLM", True),
+        ("LlamaForCausalLM", False),
     ],
 )
-def test_mtp_shared_kv_config(architecture, one_model, expected):
+def test_mtp_shared_kv_config(architecture, expected):
     spec_config = MTPDecodingConfig(
         max_draft_len=3,
         speculative_model="/tmp/assistant",
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/assistant"

(S108)